x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/ueba/host_tactics/query.host_tactics.dsl.ts

```diff
@@ -1,90 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { isEmpty } from 'lodash/fp';
-import { HostTacticsRequestOptions } from '../../../../../../common/search_strategy';
-import { createQueryFilterClauses } from '../../../../../utils/build_query';
-
-export const buildHostTacticsQuery = ({
-  defaultIndex,
-  docValueFields,
-  filterQuery,
-  hostName,
-  timerange: { from, to },
-}: HostTacticsRequestOptions) => {
-  const filter = [
-    ...createQueryFilterClauses(filterQuery),
-    {
-      range: {
-        '@timestamp': {
-          gte: from,
-          lte: to,
-          format: 'strict_date_optional_time',
-        },
-      },
-    },
-  ];
-
-  return {
-    allow_no_indices: true,
-    index: defaultIndex, // can stop getting this from sourcerer and assume default detections index if we want
-    ignore_unavailable: true,
-    track_total_hits: true,
-    body: {
-      ...(!isEmpty(docValueFields) ? { docvalue_fields: docValueFields } : {}),
-      aggs: {
```

Comment:
We might want to keep this query around until we see what the user risk score transform looks like, might be able to reuse some of this

Reply:
I had the assumption that the user risk query would look like the host risk query. Anyway, I wonder if we need to keep the file inside the codebase because it is already stored on [git history](https://github.com/elastic/kibana/blob/51856f9194fbaebebbf59e6a15782fa57fa025b4/x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/ueba/host_tactics/query.host_tactics.dsl.ts).